lib/framework/procbot-types.d.ts

```diff
@@ -40,3 +40,8 @@ export interface ConfigurationLocation {
 	 */
 	location: string | any;
 }
+
+declare const enum ProcBotErrorCode {
+	EmitterContextAbsent, NoConnectionFound, ConfigurationValueNotFound,
+	IncompleteMessage, UnsupportedMessageAction, SomethingWeird, ConversationNotFound,
```

Comment:
This is implying Messages/Conversations are part of the ProcBot framework, which they're not, they're specific to a Service implementation, so they shouldn't be in here.

I really, really don't like `SomethingWeird`. `UnknownError`, perhaps.